packages/data-point-codemods/transforms/__testfixtures__/change-path-reducer-accessing-root-path-single-quote.output.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+/* eslint-disable prettier */
+
+// prettier-ignore
+const entities = {
+  'transform:a': `$`,
+  'transform:b': `$ | foo | $`,
+  'transform:c': `foo | bar | baz`,
+  'transform:d': '$',
+  'transform:e': '$ | foo',
+  'transform:f': '$ | $ | $',
+  'transform:g': 'foo | bar | baz'
```

Comment:
should we have any tests where the `$.` is nested inside some other kind of reducer?

```js
// just for example
{
  a: ['$.']
}
```

Reply:
Sure, I can add that. Also something like `dataPoint.transform('$. | foo', input)` too?

Reply:
wouldn't hurt